pkgs/development/python-modules/numcodecs/default.nix

```diff
@@ -58,7 +58,8 @@ buildPythonPackage rec {
   ];
 
   # https://github.com/NixOS/nixpkgs/issues/255262
-  pytestFlagsArray = [ "$out/${python.sitePackages}/numcodecs" ];
+  preCheck = "pushd $out";
+  postCheck = "popd";
```

Comment:
Can be further simplified to `cd $out`.

Reply:
I suggested the `pushd; popd` approach to avoid surprises when executing subsequent phases.